simple_workflow/static/simple_workflow/rate/js/controllers.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('simple_workflow.rate.controllers')
+    .controller('ImageRatingController', ImageRatingController)
+
+  ImageRatingController.$inject = ['$scope', 'orchestraService'];
+
+  function ImageRatingController($scope, orchestraService) {
+    var vm = $scope;
+    var crawlStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'crawl');
+    if (crawlStep.task.data.status !== 'success') {
+      vm.imageURL = "http://media.giphy.com/media/2vA33ikUb0Qz6/giphy.gif";
```

Comment:
lines 12-20 of `rate/js/rate.html` describe what happens on failure. Right now it displays a warning-styled alert saying no image could be scraped, shows the funny gif, and let's the worker know that they should just submit because there's nothing to do here.

Does that sound reasonable?